module/VuFindAdmin/src/VuFindAdmin/Controller/OnlinePaymentController.php

```diff
@@ -17,8 +17,8 @@
  * GNU General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
- * along with this program; if not, see
- * <https://www.gnu.org/licenses/>.
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
```

Comment:
This has gotten accidentally reverted:

```suggestion
 * along with this program; if not, see
 * <https://www.gnu.org/licenses/>.
```

... we're going to need to figure out a way to prevent more of these old addresses from creeping back in; I expect that to be an ongoing problem for a while as PRs get merged.